core-it-suite/src/test/resources/mng-8301-project-without-modelversion/module-b/pom.xml

```
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.1.0 https://maven.apache.org/xsd/maven-4.1.0-beta-4.xsd">
+
+  <parent>
+    <groupId>org.apache.maven.its.mng8301</groupId>
+    <artifactId>m4womv</artifactId>
+  </parent>
+
+  <artifactId>module-b</artifactId>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.apache.maven.repro</groupId>
```

It’s the wrong `groupId`
```suggestion
      <groupId>org.apache.maven.its.mng8301</groupId>
```